src/Models/MetadataRelationsTrait.php

```diff
@@ -97,42 +50,43 @@ protected function getCodeForMethod(\ReflectionMethod $method) : string
         }
 
         $code = trim(preg_replace('/\s\s+/', '', $code));
-        if (false === stripos($code, 'function')) {
```

Comment:
This if check was dropped because we could not get a valid ReflectionMethod from invalid code.